app/controllers/hub/unlinked_clients_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Hub
+  class UnlinkedClientsController < ApplicationController
+    include AccessControllable
+    include ClientSortable
+
+    before_action :require_sign_in
+    before_action :load_and_authorize_national_org
+    layout "admin"
+
+    def index
+      @page_title = I18n.t("hub.clients.unlinked_clients.title")
+      @sort_column = "updated_at"
+      @sort_order = params[:order] == "desc" ? "desc" : "asc"
+      @clients = Client.where(vita_partner: @vita_partner).order(created_at: @sort_order).with_eager_loaded_associations
+    end
+
+    private
+
+    def load_and_authorize_national_org
+      @vita_partner = VitaPartner.find_or_create_national_org
```

Comment:
This view shows all of the clients that are assigned to the national organization, but those national organizations can also be assigned "valid" clients that they're needing to reassign to things. Seems like it would make more sense to _not_ assign those incoming orphans to any vita_partner and then have this view only display truly "orphaned" clients that have no vita_partner assigned at all. This might have been considered!

Reply:
I'm wondering if there are some different approaches we can take to the orphans.

- Do not assign these orphan convo clients to any vita partner
- Create a client, but do not create an intake for these new clients. Then we can target orphans by people who have no intake. (This will solve this problem, but possibly create problems down the line since we rely hardcore on every client having an intake.) 
- Do not create client/intake objects for these incoming messages at all. Instead, create a "Conversation" model that can have incoming/outgoing messages attached to non-clients. Client support can go back and forth in these interactions without a client object. Over time, we can build out logic that allows us to "convert" these conversations into client objects that save the conversation history.